docs/clusters/alpine/allocations.md

```diff
@@ -25,7 +25,7 @@ people who have overutilized their allocated resources get lower priority.
 
 ### Level Fairshare
 A Level Fairshare (`LevelFS`) is a value calculated by [Slurm's Fairshare 
-Algorithm](https://slurm.schedmd.com/fair_tree.md#algorithm). A user's 
+Algorithm](https://slurm.schedmd.com/fair_tree.html). A user's 
```

Comment:
On this page there is that 6 million number for the total number of SUs a group can have, please make sure that is correct (see other comment I made).